src/components/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -255,10 +267,37 @@ export default {
 				this.mgAmount = autoDepositAmount - this.donation;
 				this.paymentMethod = data?.my?.autoDeposit?.paymentMethod ?? {};
 				this.subscriptionId = data?.my?.autoDeposit?.id;
+				this.firstName = data?.my?.userAccount?.firstName ?? '';
+				this.subscriptionsLoans = data?.mySubscriptions?.values?.length ?? 0;
+				const mySubscriptions = data?.mySubscriptions?.values?.history?.values ?? [];
+				this.subStartTimestamp = mySubscriptions?.[mySubscriptions.length - 1]?.timeStamp ?? null;
 			}
 		},
 	},
 	computed: {
+		subStartDate() {
+			if (!this.subStartTimestamp) return '';
+			const timestamp = new Date(this.subStartTimestamp); // Replace with your own timestamp
+			return `since ${timestamp.toLocaleString('en-US', { month: 'long' })} ${timestamp.getFullYear()}!`;
+		},
+		subMonthCount() {
```

Comment:
I recommend moving these new utility methods out of the component file so that they can unit tested. Either now or in a fast follow ticket.